Author: Weab-chan
Disable has_branch_and_or since it produces inverted colors in some games
(Previously, we were reverting offending commits. In any case, this is not a permanent solution.)
(Obsolete, actually fixed by https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/28148)

diff --git a/src/freedreno/ir3/ir3_compiler.c b/src/freedreno/ir3/ir3_compiler.c
index 7e03a0a..6bc0ae1 100644
--- a/src/freedreno/ir3/ir3_compiler.c
+++ b/src/freedreno/ir3/ir3_compiler.c
@@ -218,7 +218,7 @@ ir3_compiler_create(struct fd_device *dev, const struct fd_dev_id *dev_id,
       compiler->load_inline_uniforms_via_preamble_ldgk = dev_info->a7xx.load_inline_uniforms_via_preamble_ldgk;
       compiler->num_predicates = 4;
       compiler->bitops_can_write_predicates = true;
-      compiler->has_branch_and_or = true;
+      compiler->has_branch_and_or = false;
    } else {
       compiler->max_const_pipeline = 512;
       compiler->max_const_geom = 512;
